.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
Is it possible to fix the specific  OS version we build on? For instance, is `windows-2022` available? 

Just worried about having these run non-deterministically - I presume `*-latest` means they automatically upgraded as GitHub sees fit.

Reply:
I agree, having them more deterministic would be nice, and if you feel strongly about it I can pin these to a version.

My initial thought was if there are any changes delivered in an OS update that would break Minerva, we would probably want to know about it and address it ASAP. If we hard-set these versions I wouldn't trust our ability to stay on top of updating them.

Reply:
Just something to consider, we can leave it for now and re-visit later.